src/application/Types/MovieCastDetailsTypes.ts

```diff
@@ -0,0 +1,19 @@
+export interface CrewMember {
+    job: string;
+    department: string;
+    name: string;
+    id: number;
+  }
+  
+  export interface CastMember {
+    id: number;
+    name: string;
+  }
+  
+  export interface MovieCastDetails {
+    mainCast?: CastMember[];
```

Comment:
Consistency is key here. For example in DTOs you are using Array<Foo> but here it's `Foo[]`. 

I think `Foo[]` is better. But important point here is ***consistency***.